src/Components/TruckCard/TruckCard.js

```diff
@@ -62,6 +62,8 @@ const handleUpvote = () => {
     if (!isUpvoted) {
       setupVote(upVote += 1)
       props.truck.userUpvote = true
+      // props.truck.up_rating += 1 also parse int if backend data is a string
```

Comment:
Hopefully by giving adding this value and line of code it will be easy to translate within vendor view